raytracing/specialtylenses.py

```diff
@@ -205,6 +205,156 @@ def pointsOfInterest(self, z):
         return [{'z': f1, 'label': '$F_f$'}, {'z': f2, 'label': '$F_b$'}]
 
 
+class SingletLens(MatrixGroup):
+    """
+        General singlet lens with an effective focal length of f, back focal
```

Comment:
We use the same nomenclature as Thorlabs for now. The figure is available on the product page:
https://www.thorlabs.com/images/TabImages/Plano-Convex_Lens_Schematic.gif